bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Device.java

```diff
@@ -264,6 +265,15 @@ int computePixels(float height) {
 }
 
 float computePoints(LOGFONT logFont, long hFont) {
+	return computePoints(logFont, hFont, -1);
+}
+
+float computePoints(LOGFONT logFont, long hFont, int currentFontZoomFactor) {
+	float fontDPI = DPIUtil.mapZoomToDPI(currentFontZoomFactor);
+	return computePoints(logFont, hFont, fontDPI);
+}
+
+private float computePoints(LOGFONT logFont, long hFont, float currentFontDPI) {
```

Comment:
Shouldn't `currentFontDPI` be an integer? In particular, `DPIUtil.mapZoomToDPI()`, which delivers the value passed to this method, also returns an integer.